### PR TITLE
Allow the ability to not scrub directories in the release dir

### PR DESCRIPTION
**Pull request by ConnorRigby**
To exercise this, one should create a `.noscrub` file in any
directory inside the release. any files in the same directory
will be excluded from scrubbing

### CODE REVIEW

**Comment by fhunleth** [comment]
I like how simple this is. My only comment is that it needs to be documented so that people can discover it. IMHO, the best place I can think of is in the error message right below your change. Not sure if you can think of anything better.

**Comment by ConnorRigby** [comment]
> I like how simple this is. My only comment is that it needs to be documented so that people can discover it. IMHO, the best place I can think of is in the error message right below your change. Not sure if you can think of anything better.

Yeah i sort of started making an addition to the "customizing your system" doc, but it didn't really belong there. I like your idea of the error message.

**Comment by ConnorRigby** [comment]
@fhunleth just pushed another update with the error message update.

File: scripts/scrub-otp-release.sh [hunk 1]
```diff
@@ -144,6 +151,10 @@ for EXECUTABLE in $EXECUTABLES; do
             echo "4. Are you building outside of Nerves' mix integration? If so,"
             echo "   make sure that you've sourced 'nerves-env.sh'."
             echo
+            echo "If you are very sure you know what you are doing, you may place an empty"
+            echo "file in the same directory as the offending file(s) called `.noscrub`. "
+            echo "This will explitly disable scrbbing for that directory."
```
Comment (fhunleth) [inline scripts/scrub-otp-release.sh:156]:
```suggestion
            echo "This will explicitly disable scrubbing for that directory."
```

**Review by fhunleth — APPROVED** [review]

**Pull request merged**